modules/rostests/kmtests/rtl/RtlStrSafe.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * PROJECT:         ReactOS kernel-mode tests
+ * LICENSE:         GPLv2+ - See COPYING in the top level directory
+ * PURPOSE:         Test for ntstrsafe.h functions
+ * PROGRAMMER:      Hern�n Di Pietro <hernan.di.pietro@gmail.com>
+ */
```

Comment:
Please use the new-style SPDX header.
For example: https://git.reactos.org/?p=reactos.git;a=blob;f=dll/appcompat/apphelp/apphelp.c;hb=fcf83315dccd67cd9636f9ae08868104a420a3dd#l1